data/avro-schema/src/main/resources/avro/net/corda/data/crypto/ShortHashes.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "record",
+  "name": "ShortHashes",
```

Comment:
Is this temporary code or do we want to keep both kinds of hash long term, and if so, why do we need 2?

Reply:
We will keep both hash formats, so that short forms can be used in HTTP address query strings.